test/test-variable-collector.js

```diff
@@ -49,6 +49,52 @@ describe("VariableCollector", function() {
 			"_my_other_ui_lib_MyControl_color2": "#008000"
 		}, "relevant variables should be returned");
 	});
+	it("should return relevant variables when calling 'getVariables' (special case: sap.ui.core Base/baseLib/)",
```

Comment:
As discussed: Add another test for `/resources/sap/...` paths and another where a non-core rootFilename wants to import from base